tools/cloud-build/daily-tests/builds/batch-mpi.yaml

```diff
@@ -56,7 +56,7 @@ steps:
     BUILD_ID_SHORT=$${BUILD_ID_FULL:0:6}
     SG_EXAMPLE=examples/serverless-batch-mpi.yaml
 
-    sed -i "s/# spack mirror add .*/spack mirror add --scope site gcs_cache $${SPACK_CACHE_WRF//\//\\\/}/" $${SG_EXAMPLE}
+    sed -i "s/#   spack mirror add .*/spack mirror add --scope site gcs_cache $${SPACK_CACHE_WRF//\//\\\/}/" $${SG_EXAMPLE}
```

Comment:
Just curious, why do we need the extra space?

Reply:
This sed is editing the blueprints that were updated, if you look at the change to the blueprint there is now extra space due to the indent from the if. 

The said does not bother with the if because the if is only there in the case of a reboot which will not happen in tests.